SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialClassInfoManager.h

```diff
@@ -11,13 +11,28 @@
 
 #include "SpatialClassInfoManager.generated.h"
 
-FORCEINLINE void ForAllSchemaComponentTypes(TFunction<void(ESchemaComponentType)> Callback)
+struct AllSchemaComponentTypes
```

Comment:
Nice

Reply:
This change is removed in the 0.13 branch to not overcomplicate the PR / release